geanypg/README

```diff
@@ -60,5 +60,4 @@ able to get a copy by contacting the Free Software Foundation, Inc.,
 Bugs, questions
 ---------------
 
-If you found any bugs or want to provide a patch, please contact Hans
-Alves (alves(dot)h88(at)gmail(dot)com).
+If you found any bugs or want to provide a patch, please contact Paolo Stivanin <info(at)paolostivanin(dot)com>
```

Comment:
Yes, you're right. It's better to use the github issues page. README updated with the last commit.